### PR TITLE
HomePage

### PR DESCRIPTION
**Pull request by sandy-30**

### CODE REVIEW

File: lib/pages/Home.dart [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        children:<Widget>[
+          image_carousel,
+          Expanded(
+              child:ListView.builder(
+                  scrollDirection:Axis.vertical,
+                  itemCount: 1,
+                  shrinkWrap: true,
+                  itemBuilder: (context,index) {
+                    return Classififations();
+                   }
```
Comment (sandy-30) [inline lib/pages/Home.dart]: Indent this file

File: lib/widgets/carousel.dart [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+              AssetImage('assets/images/soups/nonveg/eggsoup.jpeg'),
+              AssetImage('assets/images/soups/nonveg/manchow.jpeg'),
+            ],
+
+            autoplay: false,
+            dotSize: 4.0,
+            indicatorBgPadding: 2.0,
+            dotColor: Colors.blue,
+          ),
+
```
Comment (sandy-30) [inline lib/widgets/carousel.dart]: Avoid extra new lines

File: lib/widgets/classificationView.dart [hunk 3]
```diff
@@ -0,0 +1,63 @@
+import 'package:flutter/material.dart';
+
+class Classififations extends StatelessWidget {
+  @override
```
Comment (sandy-30) [inline lib/widgets/classificationView.dart]: Move this above build method on line 15.

File: lib/widgets/classificationView.dart [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+          padding: EdgeInsets.only(left: 10),
+          child: Container(
+            height: 230.0,
+            color: Colors.grey,
+            child: Column(children: <Widget>[
+              Row(
+                mainAxisAlignment: MainAxisAlignment.start,
+                children: <Widget>[
+                  Padding(
+                    padding: EdgeInsets.fromLTRB(10.0, 10.0, 0.0, 10.0),
```
Comment (sandy-30) [inline lib/widgets/classificationView.dart]: Re-check padding

File: lib/widgets/classificationView.dart [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+                    child: Text(
+                      'SOUPS',
+                      style: TextStyle(color: Colors.white, fontSize: 20.0),
```
Comment (sandy-30) [inline lib/widgets/classificationView.dart]: Don't introduce any styling for now.

File: lib/widgets/carousel.dart [hunk 6]
```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/material.dart';
+import 'package:carousel_pro/carousel_pro.dart';
+
+
+
+class carousels extends StatelessWidget {
```
Comment (shrkamat) [inline lib/widgets/carousel.dart]: rename carousels => Carousels

File: lib/widgets/carousel.dart [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+          height:200,
+          child: new Carousel(
+            boxFit: BoxFit.cover,
+            images: [
+              //AssetImage('images/ products/index.jpeg'),
```
Comment (shrkamat) [inline lib/widgets/carousel.dart]: Take images as argument to this widget.

File: lib/widgets/carousel.dart [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+              AssetImage('assets/images/soups/veg/mixed.jpeg'),
+              AssetImage('assets/images/soups/nonveg/creamofchicken.jpeg'),
+              AssetImage('assets/images/soups/nonveg/eggsoup.jpeg'),
+              AssetImage('assets/images/soups/nonveg/manchow.jpeg'),
+            ],
+
+            autoplay: false,
+            dotSize: 4.0,
+            indicatorBgPadding: 2.0,
+            dotColor: Colors.blue,
```
Comment (shrkamat) [inline lib/widgets/carousel.dart]: Don't update styling for now.

File: lib/widgets/classificationView.dart [hunk 9]
```diff
@@ -0,0 +1,63 @@
+import 'package:flutter/material.dart';
+
+class Classififations extends StatelessWidget {
```
Comment (shrkamat) [inline lib/widgets/classificationView.dart]: Rename Classififations => ClassififationView.

File: pubspec.lock [hunk 10]
```diff
@@ -165,3 +172,4 @@ packages:
     version: "2.1.0"
 sdks:
   dart: ">=2.12.0 <3.0.0"
+  flutter: ">=1.2.0"
```
Comment (shrkamat) [inline pubspec.lock]: Why fix flutter version ?

File: lib/pages/Home.dart [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+class MainPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+
+    Widget image_carousel=carousels();
+    return Scaffold(
+        appBar: new AppBar(
+        title: Text('RESTOSOF',style: TextStyle(fontSize:20, color: Colors.white)),
```
Comment (shrkamat) [inline lib/pages/Home.dart]: Don't apply any styling for now.

File: lib/main.dart [hunk 12]
```diff
@@ -2,13 +2,15 @@ import 'dart:async';
 import 'dart:convert';
 
 import 'package:flutter/material.dart';
+import 'package:restos/pages/Home.dart';
+import 'package:carousel_pro/carousel_pro.dart';
```
Comment (sandy-30) [inline lib/main.dart]: not required

File: lib/pages/Home.dart [hunk 13]
```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+import 'package:carousel_pro/carousel_pro.dart';
+import 'package:restos/widgets/carousel.dart';
+import 'package:restos/widgets/classificationView.dart';
+import 'package:restos/widgets/itemsView.dart';
+
+import '../main.dart';
+
+/*void main() {
```
Comment (sandy-30) [inline lib/pages/Home.dart]: Delete commented code.

File: lib/pages/Home.dart [hunk 14]
```diff
@@ -0,0 +1,42 @@
...
+          ),
+          backgroundColor: Colors.black,
+        ),
+        body: Column(
+          children: <Widget>[
+            image_carousel,
+            Expanded(
+                child: ListView.builder(
+                scrollDirection: Axis.vertical,
+                itemCount: 1,
```
Comment (sandy-30) [inline lib/pages/Home.dart]: No hardcoding data.

File: lib/pages/Home.dart [hunk 15]
```diff
@@ -0,0 +1,42 @@
...
+                shrinkWrap: true,
```
Comment (sandy-30) [inline lib/pages/Home.dart]: Why is this required ?

File: lib/pages/Home.dart [hunk 16]
```diff
@@ -0,0 +1,42 @@
...
+                itemBuilder: (context, index) {
+                return Classififations();
```
Comment (sandy-30) [inline lib/pages/Home.dart]: Check indentation

File: lib/widgets/carousel.dart [hunk 17]
```diff
@@ -0,0 +1,26 @@
...
+import 'package:carousel_pro/carousel_pro.dart';
+
+class carousels extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      height: 200,
+      child: new Carousel(
+        boxFit: BoxFit.cover,
+        images: [
```
Comment (sandy-30) [inline lib/widgets/carousel.dart]: Any data should come through constructor.

File: lib/widgets/carousel.dart [hunk 18]
```diff
@@ -0,0 +1,26 @@
...
+          AssetImage('assets/images/rest/rest1.jpeg'),
+          AssetImage('assets/images/rest/rest2.jpeg'),
+          AssetImage('assets/images/rest/rest3.jpeg'),
+          AssetImage('assets/images/rest/rest4.jpeg'),
+          AssetImage('assets/images/rest/rest5.jpeg'),
+        ],
+        autoplay: true,
+        dotSize: 4.0,
+        indicatorBgPadding: 2.0,
+        dotColor: Colors.blue,
```
Comment (sandy-30) [inline lib/widgets/carousel.dart]: Keep theams to the end.

File: lib/widgets/classificationView.dart [hunk 19]
```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/material.dart';
+import 'package:restos/widgets/itemsView.dart';
+
+class Classififations extends StatelessWidget {
+  int photoindex = 0;
+  List<String> photos = [
```
Comment (sandy-30) [inline lib/widgets/classificationView.dart]: This data should come through constructor.

File: lib/widgets/classificationView.dart [hunk 20]
```diff
@@ -0,0 +1,70 @@
...
+              Row(
+                mainAxisAlignment: MainAxisAlignment.start,
+                children: <Widget>[
+                  Padding(
+                    padding: EdgeInsets.fromLTRB(10.0, 10.0, 0.0, 10.0),
+                    child: GestureDetector(
+                      onTap: () {
+                        Navigator.push(
+                          context,
+                          MaterialPageRoute(builder: (context) => items()),
```
Comment (sandy-30) [inline lib/widgets/classificationView.dart]: Why is this required ?

File: lib/widgets/classificationView.dart [hunk 21]
```diff
@@ -0,0 +1,70 @@
...
+                        );
+                      },
+                      child: Container(
+                        child: Text(
+                          'SOUPS',
```
Comment (sandy-30) [inline lib/widgets/classificationView.dart]: Use constructor.

File: lib/widgets/itemsView.dart [hunk 22]
```diff
@@ -0,0 +1,8 @@
+import 'package:flutter/material.dart';
```
Comment (sandy-30) [inline lib/widgets/itemsView.dart]: Use this in the ListView builder here- https://github.com/shrkamat/ros/pull/1/files#diff-3ffa8ed17a5cbec6797cf9d8e357fb53e15e7197ea8100cd2041dfccfe2ec1a6R48

File: lib/widgets/carousel.dart [hunk 23]
```diff
@@ -0,0 +1,25 @@
...
+      child: new Carousel(
+        boxFit: BoxFit.cover,
+        images: [
+          AssetImage('assets/images/rest/rest1.jpeg'),
+          AssetImage('assets/images/rest/rest2.jpeg'),
+          AssetImage('assets/images/rest/rest3.jpeg'),
+          AssetImage('assets/images/rest/rest4.jpeg'),
+          AssetImage('assets/images/rest/rest5.jpeg'),
+        ],
+        /*autoplay: true,
```
Comment (shrkamat) [inline lib/widgets/carousel.dart]: delete commented code.

File: lib/widgets/carousel.dart [hunk 24]
```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
```
Comment (shrkamat) [inline lib/widgets/carousel.dart]: Delete file

File: lib/widgets/classificationView.dart [hunk 25]
```diff
@@ -0,0 +1,55 @@
...
+                          classification.name,
+                          style: TextStyle(fontSize: 20.0),
+                        ),
+                        Icon(Icons.arrow_right_sharp),
+                      ],
+                    ),
+                  ),
+                ],
+              ),
+              Column(
```
Comment (shrkamat) [inline lib/widgets/classificationView.dart]: Re-check if column is required or not.

File: lib/widgets/classificationView.dart [hunk 26]
```diff
@@ -0,0 +1,55 @@
...
+  ClassificationView(this.classification);
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(children: <Widget>[
+      Padding(
+          padding: EdgeInsets.only(left: 10),
+          child: Container(
+            height: 230.0,
+            // color: Colors.grey,
```
Comment (shrkamat) [inline lib/widgets/classificationView.dart]: Delete commented code

File: lib/widgets/classificationView.dart [hunk 27]
```diff
@@ -0,0 +1,54 @@
...
+                          style: TextStyle(
+                              fontSize: 20.0, fontWeight: FontWeight.bold),
+                        ),
+                        Icon(Icons.arrow_forward_rounded),
+                      ],
+                    ),
+                  ),
+                ],
+              ),
+              //Column(
```
Comment (shrkamat) [inline lib/widgets/classificationView.dart]: Delete commented code.

File: lib/pages/home.dart [hunk 28]
```diff
@@ -0,0 +1,42 @@
...
+  const ClassificationListView({
+    Key? key,
+    required this.classificationList,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Expanded(
+        child: ListView.builder(
+      scrollDirection: Axis.vertical,
```
Comment (shrkamat) [inline lib/pages/home.dart:35]: indentation.

File: lib/widgets/itemView.dart [hunk 29]
```diff
@@ -0,0 +1,37 @@
...
+  const ItemView(this.item, {Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      width: 140,
+      child: Column(
+        children: [
+          Container(
+            // width:140,
```
Comment (shrkamat) [inline lib/widgets/itemView.dart]: delete

File: lib/widgets/itemView.dart [hunk 30]
```diff
@@ -0,0 +1,37 @@
...
+
+            child: Card(
+              child: ClipRRect(
+                  borderRadius: BorderRadius.circular(10.0),
```
Comment (shrkamat) [inline lib/widgets/itemView.dart]: reduce to 5

File: lib/widgets/itemView.dart [hunk 31]
```diff
@@ -0,0 +1,37 @@
...
+                  child: Image.network(
+                    item.img,
+                    fit: BoxFit.fill,
+                    width: 140,
```
Comment (shrkamat) [inline lib/widgets/itemView.dart]: Make height and width constants.

File: lib/widgets/itemView.dart [hunk 32]
```diff
@@ -0,0 +1,37 @@
...
+                    height: 100,
+                  )),
+            ),
+          ),
+          SizedBox(
+              width: 120, child: Text(item.name, textAlign: TextAlign.center)),
```
Comment (shrkamat) [inline lib/widgets/itemView.dart]: width = WIDTH - 20;

**Review by shrkamat — COMMENTED** [review]
Indent all files
Delete all commented code
Delete all unused asset files
